backend/src/main/java/ai/verta/modeldb/versioning/VersioningServiceImpl.java

```diff
@@ -144,8 +144,10 @@ public void updateRepository(
     try {
       try (RequestLatencyResource latencyResource =
           new RequestLatencyResource(modelDBAuthInterceptor.getMethodName())) {
-        if (request.getRepository().getName().isEmpty()) {
-          throw new ModelDBException("Repository name is empty", Code.INVALID_ARGUMENT);
+        if (request.getRepository().getName().isEmpty()
+            && request.getRepository().getDescription().isEmpty()) {
```

Comment:
does it make an update repository request with an empty name and not empty description valid? In this case, name will be set to empty string?

Reply:
For both the questions, the answer is yes.

Reply:
this is not right , a repository can have empty description but should never have a empty name

Reply:
reverted condition for the statement `but should never have an empty name`